activation.py

```
@@ -0,0 +1,255 @@
+import json
+import time
+import argparse
+from typing import Any, Dict, Optional, Tuple, List
+
+from const import COINS_CONFIG_FILE, SCRIPT_PATH
+
+
+class CoinsConfigLoader:
+    """Loads and caches coins_config.json from the local config directory."""
+
+    _cache: Dict[str, Any] = {"ts": 0, "data": None}
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)